src/main/java/com/onevizion/scmdb/DbManager.java

```diff
@@ -258,4 +254,46 @@ public void generateDdlForAllObjects() {
         ddlGenerator.executeSettingTransformParams();
         ddlGenerator.generateDllsForAllDbObjects();
     }
+
+    /**
+     * The method filters scripts by type, separates them into two parts (arr1 and arr2).
```

Comment:
instead of describing what arr1 and arr2 means give them obvious name, and just leave info that scripts with 1..100 numbers should be the first. Also describe hot it used during development process

Reply:
Fixed.